src/actions/metadata.js

```diff
@@ -1,12 +1,16 @@
 import { getDefaultMetadata } from '../modules/metadata.js'
-import { ADD_METADATA } from '../reducers/metadata.js'
+import { ADD_METADATA, CLEAR_METADATA } from '../reducers/metadata.js'
 import { sGetRootOrgUnits } from '../reducers/settings.js'
 
 export const acAddMetadata = (value) => ({
     type: ADD_METADATA,
     value,
 })
 
+const acClearMetadata = () => ({
+    type: CLEAR_METADATA,
+})
+
 export const tSetInitMetadata = () => (dispatch, getState) => {
     const metaData = { ...getDefaultMetadata() }
```

Comment:
if `acClearMetadata` adds `DEFAULT_METADATA`. doesn't this mean that we're adding default metadata twice then with the `...getDefaultMetadata`? 🤔

Reply:
I see the confusion. CLEAR_METADATA sets the metadata to an empty object. I've renamed that constant in the reducer so it is hopefully more intuitive.